azkaban-common/src/main/java/azkaban/scheduler/Schedule.java

```diff
@@ -63,7 +68,7 @@ public Schedule(int scheduleId, int projectId, String projectName,
 
     this(scheduleId, projectId, projectName, flowName, status, firstSchedTime,
         timezone, period, lastModifyTime, nextExecTime, submitTime, submitUser,
-        null, null);
+        null, null, null);
```

Comment:
Often saw builder in other open sourced code. What is it specially focused on? Will dig one how it helps.